bundles/org.jupnp/src/main/java/org/jupnp/UpnpServiceImpl.java

```diff
@@ -85,6 +85,8 @@ public class UpnpServiceImpl implements UpnpService {
 
     protected volatile ScheduledFuture<?> scheduledFuture;
 
+    private boolean OSGiHttpServiceInjected;
```

Comment:
It's not a class name:

```suggestion
    private boolean osgiHttpServiceInjected;
```

Reply:
Removed